src/elements/content-open-with/ContentOpenWith.js

```diff
@@ -224,10 +224,10 @@ class ContentOpenWith extends PureComponent<Props, State> {
      * @return {void}
      */
     fetchOpenWithData(): void {
-        const { fileId, language }: Props = this.props;
+        const { fileId }: Props = this.props;
```

Comment:
Does `language` need to be passed in to the API constructor if we're removing it here?

Reply:
Great catch. I missed a couple of the cases where we create API instances